tests/PHPUnit/System/expected/test_reportLimiting__UserId.getUsers_day.xml

```diff
@@ -0,0 +1,183 @@
+		
+	</row>
+	<row>
+		<label>user5</label>
+		<nb_uniq_visitors>1</nb_uniq_visitors>
+		<nb_visits>5</nb_visits>
+		<nb_actions>10</nb_actions>
+		<nb_users>1</nb_users>
+		<max_actions>2</max_actions>
+		<sum_visit_length>5</sum_visit_length>
+		<bounce_count>0</bounce_count>
+		<nb_visits_converted>0</nb_visits_converted>
+		
+	</row>
+	<row>
+		<label>user6</label>
+		<nb_uniq_visitors>1</nb_uniq_visitors>
+		<nb_visits>5</nb_visits>
+		<nb_actions>10</nb_actions>
+		<nb_users>1</nb_users>
+		<max_actions>2</max_actions>
+		<sum_visit_length>5</sum_visit_length>
+		<bounce_count>0</bounce_count>
+		<nb_visits_converted>0</nb_visits_converted>
+		
+	</row>
+	<row>
+		<label>user7</label>
+		<nb_uniq_visitors>1</nb_uniq_visitors>
+		<nb_visits>5</nb_visits>
+		<nb_actions>10</nb_actions>
+		<nb_users>1</nb_users>
+		<max_actions>2</max_actions>
+		<sum_visit_length>5</sum_visit_length>
+		<bounce_count>0</bounce_count>
+		<nb_visits_converted>0</nb_visits_converted>
+		
+	</row>
+	<row>
+		<label>user8</label>
+		<nb_uniq_visitors>1</nb_uniq_visitors>
+		<nb_visits>5</nb_visits>
+		<nb_actions>10</nb_actions>
+		<nb_users>1</nb_users>
+		<max_actions>2</max_actions>
+		<sum_visit_length>5</sum_visit_length>
+		<bounce_count>0</bounce_count>
+		<nb_visits_converted>0</nb_visits_converted>
+		
+	</row>
+	<row>
+		<label>user9</label>
+		<nb_uniq_visitors>1</nb_uniq_visitors>
+		<nb_visits>5</nb_visits>
+		<nb_actions>10</nb_actions>
+		<nb_users>1</nb_users>
+		<max_actions>2</max_actions>
+		<sum_visit_length>5</sum_visit_length>
+		<bounce_count>0</bounce_count>
+		<nb_visits_converted>0</nb_visits_converted>
+		
+	</row>
+	<row>
+		<label>user10</label>
+		<nb_uniq_visitors>1</nb_uniq_visitors>
+		<nb_visits>5</nb_visits>
+		<nb_actions>10</nb_actions>
+		<nb_users>1</nb_users>
+		<max_actions>2</max_actions>
+		<sum_visit_length>5</sum_visit_length>
+		<bounce_count>0</bounce_count>
+		<nb_visits_converted>0</nb_visits_converted>
+		
+	</row>
+	<row>
+		<label>user11</label>
+		<nb_uniq_visitors>1</nb_uniq_visitors>
+		<nb_visits>5</nb_visits>
+		<nb_actions>10</nb_actions>
+		<nb_users>1</nb_users>
+		<max_actions>2</max_actions>
+		<sum_visit_length>5</sum_visit_length>
+		<bounce_count>0</bounce_count>
+		<nb_visits_converted>0</nb_visits_converted>
+		
+	</row>
+	<row>
+		<label>user12</label>
+		<nb_uniq_visitors>1</nb_uniq_visitors>
+		<nb_visits>5</nb_visits>
+		<nb_actions>10</nb_actions>
+		<nb_users>1</nb_users>
+		<max_actions>2</max_actions>
+		<sum_visit_length>5</sum_visit_length>
+		<bounce_count>0</bounce_count>
+		<nb_visits_converted>0</nb_visits_converted>
+		
+	</row>
+	<row>
+		<label>user13</label>
+		<nb_uniq_visitors>1</nb_uniq_visitors>
+		<nb_visits>5</nb_visits>
+		<nb_actions>10</nb_actions>
+		<nb_users>1</nb_users>
+		<max_actions>2</max_actions>
+		<sum_visit_length>5</sum_visit_length>
+		<bounce_count>0</bounce_count>
+		<nb_visits_converted>0</nb_visits_converted>
+		
+	</row>
+	<row>
```

Comment:
Would be useful if the tests could set a lower limit like 10, and show the behavior works (show the row `Others`)

Reply:
It does, that's the whole point of this test. See https://github.com/matomo-org/matomo/pull/14551/files#diff-5ca2964519f1aaf3fb5ca033dcb48da0